rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -151,6 +153,97 @@
             }
           }
         },
+        "frequency": {
+          "title": "Publication frequency of the issue",
+          "description": "Used for a publishing pattern that has a fundamental periodicity.",
```

Comment:
@pronguen Did you purposely specified these strings?

Reply:
This coming from the LOC MARC21 Holdings docs

Reply:
@BadrAly Okay, but this is what will be displayed in the holding editor. In my humble opinion, the title could be "Frequency", or "Publication frequency". And the description is supposed to be an help.

If it's what the PO wants for the editor, I'm fine with it. Just wanted to be sure.

Reply:
"Frequency" seems clear, understandable and short! Description is not necessary, since it is an *enum*-field.

Reply:
Done